docs/workflows.md

```diff
@@ -0,0 +1,63 @@
+## Workflows
+
+### First Steps
+
+After spinning up the containers following the `README` or `intro.md` you are ready ot rumble. Navigating to the [swagger UI](http://localhost:8000/docs/) provides a good overview of the intended functionalities.
+
+You see that tehre are `generic` endpoints for common actions, wallet specific actions and admin actions. On top of that, you find th trust registry and webhooks being exposed. These are the intended ways of client interactions with these two services.
+
+NOTE: Regardless of the multitude of containers and mechanisms running, [the CloudAPI](http://localhost:8000) and its [SwaggerUI](http://localhost:8000/docs) are the main interaction point intended between clients and the stack. This should be the only endpoint clients should (have to) interact with. There is no need (and no intention to allow that) for client to directly interact with the webhooks or trustregistry container. For a production deployment or a close-to-proudction/smoke-testing deployment, you are well adviced to only expose this endpoint to clients and leave all other endpoints unexposed to the outside world.
+
+#### Trust registry
+
+As a client you can retrieve the trust registry (and NOT alter it). That is intential and all a consumer/client should and is able to do. Altering the trust registry is baked into admin actions and only possible with admin role.
+
+#### Webhooks
+
+A client can subscribe to the webhooks via the cloudapi (as opposed to directly via the webhooks container). This requires only targetting the `/webhooks` endpoint and optionally targetting the `/webhooks/{topic}` subroute by suffixing a topic. Using the auth mechanism (see section below), the app automatically extracts the required info about the wallet (i.e the wallet id and JWT) and retrieves only the associated webhooks with a particular wallet. Failing to authenticate will return a 403 HTTP Error.
```

Comment:
How do users keep track of which webhooks they have already processed? We should mention only the last 100 webhook events can be retrieved from the api

Reply:
Ah I removed that limit after we discussed that. Reason being that before one could receive all webhooks for all wallets and all topics which broke the swagger UI. this is  not possible any more so I removed the limit (also the main use case is via code/a client and not the swagger really)